### PR TITLE
Added weekly meeting and old meeting notes

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: project/meetings.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 Meetings
 ========
 
+Weekly Meetings
+***************
+
+Every Wednedsay at 9am British Time there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.
```
Comment (Cadair) [inline project/meetings.rst:8]:
```suggestion
Every Wednedsay at 1700 UK time (1600 UTC summer, 1700 UTC winter) there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.
```

File: project/meetings.rst [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
+
+`Video Meeting <https://sunpy.org/jitsi>__`
+
+`We have a Google Calender you can import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__.
```
Comment (Cadair) [inline project/meetings.rst:12]: We could iframe this. There's also other events in the calendar, so maybe this isn't the best place?

File: project/meetings.rst [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
+Weekly Meetings
+***************
+
+Every Wednedsay at 5pm British Time there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.
+
+* `Video Meeting <https://sunpy.org/jitsi>`__
+
+* `Agenda for the meeting <https://demo.hedgedoc.org/GAEnxycXQcCQLrAFN7ie8A?both>`__
+
+* `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__
```
Comment (nabobalis) [inline project/meetings.rst:14]:
```suggestion
* `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__

.. raw:: html
<iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=America%2FLos_Angeles" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>

```

File: project/meetings.rst [hunk 4]
```diff
@@ -2,6 +2,17 @@
 Meetings
 ========
 
+Weekly Meetings
+***************
+
+Every Wednedsay at 5pm British Time there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply project/meetings.rst:8]: See if https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK works for you?

File: project/meetings.rst (hunk 2 above)
Comment (nabobalis) [reply project/meetings.rst]: Added.

File: project/meetings.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (nabobalis) [reply project/meetings.rst]: Added.

File: project/meetings.rst [hunk 5]
```diff
@@ -2,6 +2,21 @@
...
+Every Wednesday at 1700 UK time (1600 UTC summer, 1700 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK works>`__) there is a open video meeting that discusses the SunPy project and anything else that people want to cover, like a meeting or just want advice or help.
+
+* `Video Meeting <https://sunpy.org/jitsi>`__
+
+* `Agenda for the meeting <https://demo.hedgedoc.org/GAEnxycXQcCQLrAFN7ie8A?both>`__
+
+* `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__
+
+.. raw:: html
+<iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=America%2FLos_Angeles" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```
Comment (nabobalis) [inline project/meetings.rst:17]:
```suggestion
    <iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=America%2FLos_Angeles" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```

File: project/meetings.rst [hunk 6]
```diff
@@ -2,6 +2,30 @@
...
+
+    <div id="calendar-container"></div>
+
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.min.js"></script>
+    <script type="text/javascript">
+    var timezone = encodeURIComponent(jstz.determine().name());
+    var pref = '<iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=';
+    var suff = '" style=" border-width:0 " width="800" height="600" frameborder="0" scrolling="no"></iframe>';
+    var iframe_html = pref + timezone + suff;
+    document.getElementById('calendar-container').innerHTML = iframe_html;
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply project/meetings.rst:26]: I copied the code. If it works, we should merge

File: project/meetings.rst [hunk 7]
```diff
@@ -2,6 +2,30 @@
...
+.. raw:: html
+
+    <div id="calendar-container"></div>
+
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.min.js"></script>
+    <script type="text/javascript">
+    var timezone = encodeURIComponent(jstz.determine().name());
+    var pref = '<iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=';
+    var suff = '" style=" border-width:0 " width="800" height="600" frameborder="0" scrolling="no"></iframe>';
+    var iframe_html = pref + timezone + suff;
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply project/meetings.rst:25]: I guess.

File: project/meetings.rst [hunk 8]
```diff
@@ -2,6 +2,30 @@
 Meetings
 ========
 
+Weekly Meetings
+***************
+
+Every Wednesday at 1700 UK time (1600 UTC summer, 1700 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__) there is a public online meeting that discusses the SunPy project and anything else that people want to cover; like a meeting or if you require support either with Python or ``sunpy``.
+
+* `Meeting location <https://sunpy.org/jitsi>`__
```
Comment (wtbarnes) [inline project/meetings.rst:10]:
```suggestion
* `Link to join the meeting<https://sunpy.org/jitsi>`__
```

File: project/meetings.rst [hunk 9]
```diff
@@ -2,6 +2,31 @@
 Meetings
 ========
 
+Weekly Meetings
+***************
+
+Every Wednesday at 17:00 GMT+1 / 12:00 EST (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project and review open issues and code contributions.
+The meeting is open to all and we welcome discussion of any SunPy-related topics. The link to join the meeting as well as a link to the meeting agenda are included below.
```
Comment (nabobalis) [inline project/meetings.rst:9]:
```suggestion
The meeting is open to all and we welcome discussion of any SunPy-related topics.
The link to join the meeting as well as a link to the meeting agenda are included below.
```

File: project/meetings.rst [hunk 10]
```diff
@@ -2,6 +2,32 @@
...
 ========
 
+Weekly Meetings
+***************
+
+Every Wednesday at 17:00 GMT+1 / 12:00 EST (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project and review open issues and code contributions.
+The meeting is open to all and we welcome discussion of any SunPy-related topics.
+The link to join the meeting as well as a link to the meeting agenda are included below.
+
+* `Link to join the meeting<https://sunpy.org/jitsi>`__
```
Comment (nabobalis) [inline project/meetings.rst:12]:
```suggestion
* `Link to join the meeting <https://sunpy.org/jitsi>`__
```

File: project/meetings.rst [hunk 11]
```diff
@@ -2,6 +2,30 @@
...
+
+Every Wednesday at 1700 UK time (1600 UTC summer, 1700 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__) there is a public online meeting that discusses the SunPy project and anything else that people want to cover; like a meeting or if you require support either with Python or ``sunpy``.
+
+* `Meeting location <https://sunpy.org/jitsi>`__
+
+* `Agenda for the meeting <https://demo.hedgedoc.org/GAEnxycXQcCQLrAFN7ie8A?both>`__
+
+* `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__
+
+.. raw:: html
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply project/meetings.rst:18]: Will do

[the remark this answers is not in the document]
Comment (nabobalis) [reply project/meetings.rst:5]: Done.

File: project/meetings.rst [hunk 12]
```diff
@@ -2,6 +2,32 @@
...
+    var pref = '<iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=';
+    var suff = '" style=" border-width:0 " width="800" height="600" frameborder="0" scrolling="no"></iframe>';
+    var iframe_html = pref + timezone + suff;
+    document.getElementById('calendar-container').innerHTML = iframe_html;
+    </script>
+
+Weekly Meetings
+***************
+
+Every Wednesday at 17:00 GMT+1 / 12:00 EST (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project and review open issues and code contributions.
```
Comment (Cadair) [inline project/meetings.rst:21]: This edit isn't correct, the time actually changes during the year, the only "accurate" way to say it is that it happens at 1700 London time.

```suggestion
Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project and review open issues and code contributions.
```

File: project/meetings.rst [hunk 13]
```diff
@@ -2,6 +2,32 @@
...
+The meeting is open to all and we welcome discussion of any SunPy-related topics.
```
Comment (Cadair) [inline project/meetings.rst:22]: We just said a much more limited description of the meeting topic above and now we are saying we are open to talking about anything.

File: project/meetings.rst (hunk 12 above)
Comment (Cadair) [inline project/meetings.rst:21]:
```suggestion
Every Wednesday at 17:00 GMT+1 / 12:00 EST (16:00 UTC summer, 17:00 UTC winter, `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting to discuss items related to the project such as reviewing open issues and code contributions.
```

File: project/meetings.rst (hunk 6 above)
[the remark this answers is not in the document]
Comment (Cadair) [reply project/meetings.rst:15]: It's so ugly even I can tell it's nasty js.

File: project/meetings.rst [hunk 14]
```diff
@@ -2,6 +2,32 @@
...
+    <div id="calendar-container"></div>
+
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.min.js"></script>
+    <script type="text/javascript">
+    var timezone = encodeURIComponent(jstz.determine().name());
+    var pref = '<iframe src="https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=';
+    var suff = '" style=" border-width:0 " width="800" height="600" frameborder="0" scrolling="no"></iframe>';
+    var iframe_html = pref + timezone + suff;
+    document.getElementById('calendar-container').innerHTML = iframe_html;
+    </script>
```
Comment (Cadair) [inline project/meetings.rst:16]:
```suggestion
    <iframe id="calendar" src="" width="800" height="600" frameborder="0" scrolling="no"></iframe>
    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jstimezonedetect/1.0.4/jstz.min.js"></script>
    <script type="text/javascript">
    var timezone = encodeURIComponent(jstz.determine().name());
    var iframe_url = `https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=${timezone}`;
    document.getElementById('calendar').src = iframe_url;
    </script>
```

File: project/meetings.rst [hunk 15]
```diff
@@ -2,6 +2,29 @@
...
+    <script type="text/javascript">
+    var timezone = encodeURIComponent(jstz.determine().name());
+    var iframe_url = `https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=${timezone}`;
+    document.getElementById('calendar').src = iframe_url;
+    </script>
+
+Weekly Meetings
+***************
+
+Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter) `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__, we hold a community meeting to discuss items related to the project and review open issues and code contributions.
```
Comment (nabobalis) [inline project/meetings.rst:18]:
```suggestion
Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter) `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__, we hold a community meeting.
```

File: project/meetings.rst (hunk 13 above)
Comment (nabobalis) [reply project/meetings.rst:19]: Not anymore.

File: project/meetings.rst [hunk 16]
```diff
@@ -2,6 +2,29 @@
...
+    <script type="text/javascript">
+    var timezone = encodeURIComponent(jstz.determine().name());
+    var iframe_url = `https://calendar.google.com/calendar/embed?src=g9c9eakg98b5cbogd7m5ta6h8s%40group.calendar.google.com&ctz=${timezone}`;
+    document.getElementById('calendar').src = iframe_url;
+    </script>
+
+Weekly Meetings
+***************
+
+Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter) `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__, we hold a community meeting.
```
Comment (nabobalis) [inline project/meetings.rst:18]:
```suggestion
Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter, you can use this `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting.
```

File: project/meetings.rst [hunk 17]
```diff
@@ -2,6 +2,29 @@
...
+
+Every Wednesday at 17:00 London time (16:00 UTC summer, 17:00 UTC winter, you can use this `timezone convertor <https://dateful.com/time-zone-converter?t=5pm&tz2=London-UK>`__), we hold a community meeting.
+The meeting is open to all and we welcome discussion of any SunPy-related topics.
+The link to join the meeting as well as a link to the meeting agenda are included below.
+
+* `Link to join the meeting <https://sunpy.org/jitsi>`__
+
+* `Agenda for the meeting <https://demo.hedgedoc.org/GAEnxycXQcCQLrAFN7ie8A?both>`__
+
+* `Google Calender import <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__
```
Comment (Cadair) [inline project/meetings.rst:26]:
```suggestion
* `Google Calender <https://calendar.google.com/calendar/u/0?cid=ZzljOWVha2c5OGI1Y2JvZ2Q3bTV0YTZoOHNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ>`__
```

File: project/meetings.rst [hunk 18]
```diff
@@ -50,3 +72,9 @@ Some suggested places to stay, please note there is no official affiliation with
 * `Clayton Hotel Burlington Road <https://bookings.claytonhotelburlingtonroad.com>`_, Leeson Street Upper, Dublin, D04 A318
 * `Mespil Hotel <https://www.mespilhotel.com>`_, 50-60 Mespil Rd, Dublin 4, D04 E7N2
 * `Pembroke Townhouse <https://www.pembroketownhouse.ie>`_, 88 Pembroke Rd, Dublin, D04 TF72
+
+Previous SunPy Coordination Meeting
+***********************************
```
Comment (Cadair) [inline project/meetings.rst:77]:
```suggestion
Previous SunPy Coordination Meetings
************************************
```

**Review by Cadair — APPROVED** [review]

**Pull request merged**